Initial build for QA

clpetersonucf opened the pull request.

FrenjaminBanklin commented:
Wouldn't delay deploying because of it, but these inputs are a bit short considering the wide open space to the right of them.

![lots-o-white-space](https://user-images.githubusercontent.com/1275983/51331403-ac772380-1a47-11e9-9f5b-a2cdd95ac92a.png)

iturgeon commented on src/install.yaml:
```diff
@@ -1,6 +1,6 @@
 ---
 general:
```
Go ahead and make this widget visible in the catalog by default

iturgeon commented:
I'm getting a `https://code.angularjs.org/1.6.9/docs/error/ngRepeat/dupes?p0=response%20in%20responses&p1=string:Sometimes&p2=Sometimes` error on the score screen

iturgeon commented:
One more feedback item - I think the title is a little hard to notice.  Chances are people will miss it all together. 

Perhaps in a future phase we should do the little setup card like Labeling

iturgeon commented:
Scratch that last emoji title issue - it's a Materia issue https://github.com/ucfopen/Materia/issues/909

clpetersonucf commented:
Implemented the following corrections based on feedback:
- Added demo
- set `in_catalog` flag to `true`
- Expanded min-width of certain inputs
- Removed default question text and added placeholder text instead
- Fixed issue with scorescreen choking on duplicate responses in responses array
- onQuestionImportComplete should be working properly now

iturgeon commented:
Importing error

`Uncaught TypeError: Cannot read property 'questionType' of undefined`


```$scope.onQuestionImportComplete = function(items) {
      var i, item, len1;
      for (i = 0, len1 = items.length; i < len1; i++) {
        item = items[i];
        $scope.cards.push({
          question: item.questions[0].text,
          questionType: item.options.questionType,
          answerType: item.options.answerType,
          answers: item.answers,
          displayStyle: item.options.displayStyle,
          group: item.options.group
        });
        questionCount++;
      }
      return $scope.$apply(function() {
        return $scope.showToast("Added " + items.length + " imported questions.");
      });
    };```

`questionType: item.options.questionType,` seems to be an issue in the mwdk anyway

nathaned commented:
![image](https://user-images.githubusercontent.com/10188094/51405397-38f71400-1b24-11e9-8b82-9e50c815d757.png)

feedback on scorescreen:

- I think there should be some type of "your response has been recorded" for the survey. maybe a big banner at the top with a check
- Icons in top right were confusing when everything was MC. Once I saw the demo with the checkbox and the edit pen it made more sense, but I don't think they're really necessary

Also, in the creator, messages from `Materia.CreatorCore.cancelSave` are concatenated to "Unfortunately your progress was not saved because", so those should be reworded.
